facia/app/controllers/FaciaController.scala

```diff
@@ -125,23 +125,23 @@ trait FaciaController extends BaseController with Logging with ImplicitControlle
 
   import PressedPage.pressedPageFormat
   private[controllers] def renderFrontPressResult(path: String)(implicit request: RequestHeader) = {
-    val futureFaciaPage: Future[Option[PressedPage]] = frontJsonFapi.get(path, liteRequestType).flatMap {
+    val futureFaciaPage: Future[Option[(PressedPage, Boolean)]] = frontJsonFapi.get(path, liteRequestType).flatMap {
```

Comment:
use a nasty tuple so that we can store whether the page initially contained targeted containers (even if they are later removed)